### PR TITLE
fix: make "Manage Reusable Blocks" a link

### LINKED ISSUE #10009: Reusable blocks: the "Manage all..." link in the Inserter should look like a link
When there are saved reusable blocks, a link that looks like a button is displayed in the inserter, in the Reusable block section:

screen shot 2018-09-18 at 15 46 52

This is a link styled like a button. It just happened to me, using the keyboard, to press spacebar because it looks like a button and I've stared at the screen waiting for something to happen...
Of course, nothing happened because it's a link and it gets activated just with the Enter key.

This is a very good example of a design choice made just for visual purposes that is actually confusing for users. This is a link, it triggers navigation to an actual, different admin page (`wp-admin/edit.php?post_type=wp_block`) thus should look like a link.

Other improvements are certainly possible. For example, the discoverability of this control could be improved. Also, seems to me there's a bit of excessive title-casing: `Manage All Reusable Blocks` seems a bit too much to me.

For reference, see #8888 / #8927, and #4325 / #4471. See also the copy guidelines https://github.com/WordPress/gutenberg/blob/master/docs/reference/copy-guide.md#five-pay-attention-to-capitalization 

Seems there's still a bit of inconsistency across the UI:

screen shot 2018-09-18 at 15 47 33
screen shot 2018-09-18 at 15 47 39
screen shot 2018-09-18 at 15 47 41
screen shot 2018-09-18 at 15 47 42

### PR DESCRIPTION
**Pull request by tofumatt**
Fix #10009.

## Description
Convert the "Manage all Reusable Blocks" button to a link. Previously, it was an icon-only button, which isn't good for accessibility or UX, and caused confusion regarding the result of interacting with the button (see: #10435).

This makes it clear the item is a link which will navigate to another part of the site.

## How has this been tested?
Tested locally in Firefox and Chrome, link works as expected. No longer looks like a button that doesn't act like one.

### Before
screenshot 2018-10-09 21 23 01
screenshot 2018-10-09 21 22 52

### After
screenshot 2018-10-10 00 21 59

### CODE REVIEW

**Comment by tofumatt** [comment]
I centered the text because I thought it was nice. Feel free to adjust that 😉

**Review by noisysocks — APPROVED** [review]
Tests well and looks good from a code perspective 👍

**Comment by jasmussen** [comment]
I think the location of the link in the inserter is something that, in the future, should probably be reconsidered. The block library feels like it should contain only blocks, like a drawer that is intentionally created to hold a specific type of items, and when it contains a link like this it feels weird like an odd thing that "sticks out", especially given we also have a link in the menu. However it's also probably necessary there, because users might discover that link from the reusable blocks section.

In other words, this is fine, ship it. But it'd be nice if the reusable blocks management UI could see improvement separately in the future. But such it is with software, work is never over, and perfection is always around the next corner. Good thing 5.1 is already slated for early 2019.

Nice work.

**Comment by tofumatt** [comment]
Agreed 100%. This is still a pretty janky UX, but at least the bank is well-communicated with it being a link now. 😆

**Comment by jasmussen** [comment]
> Do we need both this link and the Mange All Reusable Blocks menu item?

That's the million dollar question. 

My gut says _yes_ because reusable blocks are a new concept, and we can speculate a user might intuitively want to look for options in the _block library_. Also, redundant interfaces is not the end of the world. 

But it would be nice to remove it if we had some way to know that the More menu link was sufficient.

**Comment by tofumatt** [comment]
WordPress.com Gutenberg usage stats could probably tell us that, but having the manage link right next to the items _seems_ nice; it’s just a shame they link elsewhere, but that’s another issue really—building a way to mange them within the editor is a much bigger task.

- Matt (Sent from mobile)
